src/main/java/de/zalando/aruha/nakadi/repository/kafka/KafkaTopicRepository.java

```diff
@@ -58,7 +58,7 @@ public KafkaRepository(final ZooKeeperHolder zkFactory, final KafkaFactory kafka
                     .map(Topic::new)
                     .collect(Collectors.toList());
         } catch (Exception e) {
```

Comment:
I agree. But unfortunately the Curator Framework isn't more specific as well and doesn't expose something like a ZooKeeperException, which would be nice. :(
If you look into the `forPath()`, it only declares a generic exception in the signature - unfortunately.